_posts/2017-03-06-new-topology-system.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: A shiny, new topology system 
+---
+
+With MDAnalysis 0.16.0 on the horizon, we wanted to showcase a major development that most users will probably not notice if we've done our job well.
+In fall 2015, @richardjgowers and I set to work on redesigning the topology system from scratch.
```

Comment:
I'm still a little amazed at how close to our first picture (I think this is a tidied version though) we managed to get